gitlab/integration.sh

```diff
@@ -81,11 +81,26 @@ mount
 mount -o remount,exec,dev /builds
 section_end mount
 
+
+section_start check_cgroup_v1 "Checking for cgroup v1 availability"
+grep cgroup$ /proc/filesystems
+if [ $? -eq 0 ]; then
+    cgroupv1=1
+else
+    echo "Skipping tests that rely on cgroup v1"
+    cgroupv1=0
+fi
+section_end check_cgroup_v1
+
 section_start judgehost "Configure judgehost"
 cd /opt/domjudge/judgehost/
 sudo cp /opt/domjudge/judgehost/etc/sudoers-domjudge /etc/sudoers.d/
 sudo chmod 400 /etc/sudoers.d/sudoers-domjudge
-sudo bin/create_cgroups
+if [ $cgroupv1 -ne 0 ]; then
+    # We allow this to go wrong as some gitlab runners do not have the
+    # swapaccount kernel option set.
+    sudo bin/create_cgroups || cgroupv1=0
```

Comment:
```suggestion
    sudo bin/create_cgroups || cgroupv=0
```
The choice for cgroupv=0 is explicit here but I would argue that we should fail for this case? If the gitlab runners are inconsistent I would prefer that we fail get a retry on another runner. 

Why can this fail on a machine with cgroupv1?